test/src/worker/browser-env/runner/test-runner/index.ts

```diff
@@ -372,41 +380,87 @@ describe("worker/browser-env/runner/test-runner", () => {
         });
 
         describe(`"${BrowserEventNames.runBrowserCommand}" event`, () => {
-            describe("should return error as first callback argument if", () => {
-                it("command does not exists in browser instance", done => {
-                    const browser = mkBrowser_();
-                    (BrowserAgent.prototype.getBrowser as SinonStub).resolves(browser);
-                    const socket = mkSocket_() as BrowserViteSocket;
-                    socketClientStub.returns(socket);
+            describe("call command on element instance", () => {
```

Comment:
I tried to combine the tests for the element and for the browser, but the code turns out to be too complicated